File: unittest/scripts/auto/py_devapi/scripts/mysqlx_sqlresult_help.py
shell.connect(__uripwd + '/mysql')
sqlresult = session.sql('select 1;').execute()
session.close()

#@ sqlresult
sqlresult.help()

#@ global ? for SqlResult[USE:sqlresult]
\? SqlResult

#@ global help for SqlResult[USE:sqlresult]
\help SqlResult

#@ sqlresult.affected_items_count
sqlresult.help('affected_items_count')

#@ global ? for affected_items_count[USE:sqlresult.affected_items_count]
\? SqlResult.affected_items_count

#@ global help for affected_items_count[USE:sqlresult.affected_items_count]
\help SqlResult.affected_items_count

#@ sqlresult.auto_increment_value
sqlresult.help('auto_increment_value')

#@ global ? for auto_increment_value[USE:sqlresult.auto_increment_value]
\? SqlResult.auto_increment_value

#@ global help for auto_increment_value[USE:sqlresult.auto_increment_value]
\help SqlResult.auto_increment_value

#@ sqlresult.column_count
sqlresult.help('column_count')

#@ global ? for column_count[USE:sqlresult.column_count]
\? SqlResult.column_count

#@ global help for column_count[USE:sqlresult.column_count]
\help SqlResult.column_count

#@ sqlresult.column_names
sqlresult.help('column_names')

#@ global ? for column_names[USE:sqlresult.column_names]
\? SqlResult.column_names

#@ global help for column_names[USE:sqlresult.column_names]
\help SqlResult.column_names

#@ sqlresult.columns
sqlresult.help('columns')

#@ global ? for columns[USE:sqlresult.columns]
\? SqlResult.columns

#@ global help for columns[USE:sqlresult.columns]
\help SqlResult.columns

#@ sqlresult.execution_time
sqlresult.help('execution_time')

#@ global ? for execution_time[USE:sqlresult.execution_time]
\? SqlResult.execution_time

#@ global help for execution_time[USE:sqlresult.execution_time]
\help SqlResult.execution_time

#@ sqlresult.fetch_all
sqlresult.help('fetch_all')

#@ global ? for fetch_all[USE:sqlresult.fetch_all]
\? SqlResult.fetch_all

#@ global help for fetch_all[USE:sqlresult.fetch_all]
\help SqlResult.fetch_all

#@ sqlresult.fetch_one
sqlresult.help('fetch_one')

#@ global ? for fetch_one[USE:sqlresult.fetch_one]
\? SqlResult.fetch_one

#@ global help for fetch_one[USE:sqlresult.fetch_one]
\help SqlResult.fetch_one

#@ sqlresult.fetch_one_object
sqlresult.help('fetch_one_object')

#@ global ? for fetch_one_object[USE:sqlresult.fetch_one_object]
\? SqlResult.fetch_one_object

#@ global help for fetch_one_object[USE:sqlresult.fetch_one_object]
\help SqlResult.fetch_one_object

#@ sqlresult.get_affected_items_count
sqlresult.help('get_affected_items_count')

#@ global ? for get_affected_items_count[USE:sqlresult.get_affected_items_count]
\? SqlResult.get_affected_items_count

#@ global help for get_affected_items_count[USE:sqlresult.get_affected_items_count]
\help SqlResult.get_affected_items_count

#@ sqlresult.get_auto_increment_value
sqlresult.help('get_auto_increment_value')

#@ global ? for get_auto_increment_value[USE:sqlresult.get_auto_increment_value]
\? SqlResult.get_auto_increment_value

#@ global help for get_auto_increment_value[USE:sqlresult.get_auto_increment_value]
\help SqlResult.get_auto_increment_value

#@ sqlresult.get_column_count
sqlresult.help('get_column_count')

#@ global ? for get_column_count[USE:sqlresult.get_column_count]
\? SqlResult.get_column_count

#@ global help for get_column_count[USE:sqlresult.get_column_count]
\help SqlResult.get_column_count

#@ sqlresult.get_column_names
sqlresult.help('get_column_names')

#@ global ? for get_column_names[USE:sqlresult.get_column_names]
\? SqlResult.get_column_names

#@ global help for get_column_names[USE:sqlresult.get_column_names]
\help SqlResult.get_column_names

#@ sqlresult.get_columns
sqlresult.help('get_columns')

#@ global ? for get_columns[USE:sqlresult.get_columns]
\? SqlResult.get_columns

#@ global help for get_columns[USE:sqlresult.get_columns]
\help SqlResult.get_columns

#@ sqlresult.get_execution_time
sqlresult.help('get_execution_time')

#@ global ? for get_execution_time[USE:sqlresult.get_execution_time]
\? SqlResult.get_execution_time

#@ global help for get_execution_time[USE:sqlresult.get_execution_time]
\help SqlResult.get_execution_time

#@ sqlresult.get_warnings
sqlresult.help('get_warnings')

#@ global ? for get_warnings[USE:sqlresult.get_warnings]
\? SqlResult.get_warnings

#@ global help for get_warnings[USE:sqlresult.get_warnings]
\help SqlResult.get_warnings

#@ sqlresult.get_warnings_count
sqlresult.help('get_warnings_count')

#@ global ? for get_warnings_count[USE:sqlresult.get_warnings_count]
\? SqlResult.get_warnings_count

#@ global help for get_warnings_count[USE:sqlresult.get_warnings_count]
\help SqlResult.get_warnings_count

#@ sqlresult.has_data
sqlresult.help('has_data')

#@ global ? for has_data[USE:sqlresult.has_data]
\? SqlResult.has_data

#@ global help for has_data[USE:sqlresult.has_data]
\help SqlResult.has_data

#@ sqlresult.help
sqlresult.help('help')

#@ global ? for help[USE:sqlresult.help]
\? SqlResult.help

#@ global help for help[USE:sqlresult.help]
\help SqlResult.help

#@ sqlresult.next_result
sqlresult.help('next_result')

#@ global ? for next_result[USE:sqlresult.next_result]
\? SqlResult.next_result

#@ global help for next_result[USE:sqlresult.next_result]
\help SqlResult.next_result

#@ sqlresult.warnings
sqlresult.help('warnings')

#@ global ? for warnings[USE:sqlresult.warnings]
\? SqlResult.warnings

#@ global help for warnings[USE:sqlresult.warnings]
\help SqlResult.warnings

#@ sqlresult.warnings_count
sqlresult.help('warnings_count')

#@ global ? for warnings_count[USE:sqlresult.warnings_count]
\? SqlResult.warnings_count

#@ global help for warnings_count[USE:sqlresult.warnings_count]
\help SqlResult.warnings_count

File: unittest/scripts/auto/js_devapi/scripts/mysqlx_sqlresult_help.js
//@ Initialization
shell.connect(__uripwd);
var result = session.sql("select 1").execute();
session.close();

//@ SqlResult help
result.help();

//@ SqlResult help, \? [USE:SqlResult help]
\? SqlResult

//@ Help on affectedItemsCount
result.help('affectedItemsCount');

//@ Help on affectedItemsCount, \? [USE:Help on affectedItemsCount]
\? SqlResult.affectedItemsCount

//@ Help on affectedRowCount
result.help('affectedRowCount');

//@ Help on affectedRowCount, \? [USE:Help on affectedRowCount]
\? SqlResult.affectedRowCount

//@ Help on autoIncrementValue
result.help('autoIncrementValue');

//@ Help on autoIncrementValue, \? [USE:Help on autoIncrementValue]
\? SqlResult.autoIncrementValue

//@ Help on columnCount
result.help('columnCount');

//@ Help on columnCount, \? [USE:Help on columnCount]
\? SqlResult.columnCount

//@ Help on columnNames
result.help('columnNames');

//@ Help on columnNames, \? [USE:Help on columnNames]
\? SqlResult.columnNames

//@ Help on columns
result.help('columns');

//@ Help on columns, \? [USE:Help on columns]
\? SqlResult.columns

//@ Help on executionTime
result.help('executionTime');

//@ Help on executionTime, \? [USE:Help on executionTime]
\? SqlResult.executionTime

//@ Help on warningCount
result.help('warningCount');

//@ Help on warningCount, \? [USE:Help on warningCount]
\? SqlResult.warningCount

//@ Help on warnings
result.help('warnings');

//@ Help on warnings, \? [USE:Help on warnings]
\? SqlResult.warnings

//@ Help on warningsCount
result.help('warningsCount');

//@ Help on warningsCount, \? [USE:Help on warningsCount]
\? SqlResult.warningsCount

//@ Help on fetchAll
result.help('fetchAll');

//@ Help on fetchAll, \? [USE:Help on fetchAll]
\? SqlResult.fetchAll

//@ Help on fetchOne
result.help('fetchOne');

//@ Help on fetchOne, \? [USE:Help on fetchOne]
\? SqlResult.fetchOne

//@ Help on fetchOneObject
result.help('fetchOneObject');

//@ Help on fetchOneObject, \? [USE:Help on fetchOneObject]
\? SqlResult.fetchOneObject

//@ Help on getAffectedItemsCount
result.help('getAffectedItemsCount');

//@ Help on getAffectedItemsCount, \? [USE:Help on getAffectedItemsCount]
\? SqlResult.getAffectedItemsCount

//@ Help on getAffectedRowCount
result.help('getAffectedRowCount');

//@ Help on getAffectedRowCount, \? [USE:Help on getAffectedRowCount]
\? SqlResult.getAffectedRowCount

//@ Help on getAutoIncrementValue
result.help('getAutoIncrementValue');

//@ Help on getAutoIncrementValue, \? [USE:Help on getAutoIncrementValue]
\? SqlResult.getAutoIncrementValue

//@ Help on getColumnCount
result.help('getColumnCount');

//@ Help on getColumnCount, \? [USE:Help on getColumnCount]
\? SqlResult.getColumnCount

//@ Help on getColumnNames
result.help('getColumnNames');

//@ Help on getColumnNames, \? [USE:Help on getColumnNames]
\? SqlResult.getColumnNames

//@ Help on getColumns
result.help('getColumns');

//@ Help on getColumns, \? [USE:Help on getColumns]
\? SqlResult.getColumns

//@ Help on getExecutionTime
result.help('getExecutionTime');

//@ Help on getExecutionTime, \? [USE:Help on getExecutionTime]
\? SqlResult.getExecutionTime

//@ Help on getWarningCount
result.help('getWarningCount');

//@ Help on getWarningCount, \? [USE:Help on getWarningCount]
\? SqlResult.getWarningCount

//@ Help on getWarnings
result.help('getWarnings');

//@ Help on getWarnings, \? [USE:Help on getWarnings]
\? SqlResult.getWarnings

//@ Help on getWarningsCount
result.help('getWarningsCount');

//@ Help on getWarningsCount, \? [USE:Help on getWarningsCount]
\? SqlResult.getWarningsCount

//@ Help on hasData
result.help('hasData');

//@ Help on hasData, \? [USE:Help on hasData]
\? SqlResult.hasData

//@ Help on help
result.help('help');

//@ Help on help, \? [USE:Help on help]
\? SqlResult.help

//@ Help on nextDataSet
result.help('nextDataSet');

//@ Help on nextDataSet, \? [USE:Help on nextDataSet]
\? SqlResult.nextDataSet
